### ISSUE TITLE
Error installing on Ubuntu 20.04: t-rex depends on json-c-0.13.1 (>= 20180305)

### ISSUE THREAD

**Issue by tomasholderness** [t-rex-tileserver/t-rex#212]
Hi,

installing the 't-rex_0.11.0_amd64_focal.deb' on Ubuntu 20.04 gives the following error. Is this looking for libjson-c-dev? (currently installed: 0.13.1+dfsg-7ubuntu0.3).

thanks.

```sh
sudo dpkg -i t-rex_0.11.0_amd64_focal.deb 
(Reading database ... 61817 files and directories currently installed.)
Preparing to unpack t-rex_0.11.0_amd64_focal.deb ...
Unpacking t-rex (0.11.0) over (0.11.0) ...
dpkg: dependency problems prevent configuration of t-rex:
 t-rex depends on json-c-0.13.1 (>= 20180305); however:
  Package json-c-0.13.1 is not installed.

dpkg: error processing package t-rex (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 t-rex
```

**Comment by pka** [comment]
No idea why this dependency slipped in. The current workaround is installing it with
```
sudo dpkg --ignore-depends=json-c -i t-rex_0.11.0_amd64_focal.deb
```

**Comment by pka** [comment]
Has been fixed in 0.12.0
